### PR TITLE
Convert developer download, first run and whats new pages to Protocol

### LINKED ISSUE #7485: Convert /firefox/developer to Protocol and update design
New design, no string changes:

fx-dev-edi-1312

This will also impact the first run and whats new pages since they share a common style sheet.

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

- Convert firefox/developer to Protocol
- Convert firefox/developer/<67/whatsnew to Protocol
- Convert firefox/developer/>57/firstrun to Protocol
- Add newsletter JS to all pages
- Move from 3 seperate CSS bundles to one
- Standardize design elements across all 3 pages
- Remove old design CSS and assets
- Add hidden language field to newsletter signup form if newsletter is English only.

## Issue / Bugzilla link

Fix #7485

## Testing
- http://localhost:8000/en-US/firefox/developer/
- http://localhost:8000/en-US/firefox/67.0a2/whatsnew/
- http://localhost:8000/en-US/firefox/67.0a2/firstrun/

### CODE REVIEW

**Comment by craigcook** [comment]
Waiting for some design changes

**Comment by stephaniehobson** [comment]
Might still be one small design change coming but it will be small so this is ready for review.

File: bedrock/firefox/templates/firefox/developer/index.html [hunk 1]
```diff
@@ -29,67 +29,68 @@
...
-          <a href="{{ url('privacy.notices.firefox') }}#telemetry" class="more">{{ _('Learn more') }}</a>
-        </p>
-      </div>
-
-      {{ high_res_img('firefox/developer/hero-screenshot.png', {'class': 'hero-image', 'alt': '', 'width': '1200', 'height': '400'}) }}
+      {{ download_firefox('alpha', platform='desktop', dom_id='intro-download', button_color='button-arrow', download_location='primary cta') }}
 
+      <p class="intro-feedback-note">
+        {{ _('Firefox Developer Edition automatically sends feedback to Mozilla.') }}
+        <a href="{{ url('privacy.notices.firefox') }}#telemetry" class="more">{{ _('Learn more') }}</a>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/developer/index.html]: The section this links to is closed in the updated privacy page. I'm not sure there's anything we can do about that within the scope of this PR but perhaps at least we can link to `#health-report` instead (that's the parent section containing the telemetry stuff).

File: bedrock/firefox/templates/firefox/developer/includes/highlights.html [hunk 2]
```diff
@@ -1,103 +1,88 @@
...
-      <div class="section-body">
-        <a rel="external" href="https://developer.mozilla.org/en-US/docs/Tools/Page_Inspector/How_to/Edit_fonts">
-          <h3 class="body-title">{{ _('Fonts Panel') }}</h3>
-
-          <img class="hero-image" src="{{ static('img/firefox/developer/hero-fonts.gif') }}" alt="">
-        </a>
-        <p>
+    {% if l10n_has_tag('new_features_20180830') %}
+    <div class="c-gallery-item">
+      <a rel="external" href="https://developer.mozilla.org/en-US/docs/Tools/Page_Inspector/How_to/Edit_CSS_shapes">
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/developer/includes/highlights.html]: ✅

File: bedrock/firefox/templates/firefox/developer/includes/highlights.html [hunk 3]
```diff
@@ -1,103 +1,88 @@
...
-        </a>
-        <p>
+<section class="t-highlights mzp-l-content">
+  <div class="c-gallery-grid">
+    <div class="c-gallery-item">
+      <a href="https://mozilladevelopers.github.io/playground/debugger/" rel="external">
+        <img class="highlight-image" src="{{ static('img/firefox/developer/hero-debugger-ani.gif') }}" alt="_('Firefox DevTools')">
+      </a>
+      <h2 class="c-title">{{ _('New Tools') }}</h2>
+      <h3 id="devtools" class="c-subtitle">{{ _('Firefox DevTools') }}</h3>
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/developer/includes/highlights.html]: ✅

File: bedrock/firefox/templates/firefox/developer/includes/intro-features.html [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
+    <h3 class="c-title">{{ _('Master CSS Grid') }}</h3>
+    <p><a href="#cssgrid" class="mzp-c-cta-link">{{ _('Learn more') }}</a></p>
   </li>
-  <li>
+  <li class="c-gallery-item">
     <img class="icon" src="{{ static('img/firefox/developer/next-gen-icon.svg') }}" alt="">
-    <h3>{{ _('Next-Gen CSS Engine') }}</h3>
-    <p class="cta"><a href="#nextgen" class="button button-hollow">{{ _('Learn more') }}</a></p>
+    <h3 class="c-title">{{ _('Next-Gen CSS Engine') }}</h3>
+    <p><a href="#nextgen" class="mzp-c-cta-link">{{ _('Learn more') }}</a></p>
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/developer/includes/intro-features.html]: ✅

File: bedrock/firefox/templates/firefox/developer/includes/performance.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
+<section class="mzp-l-content t-performance">
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/developer/includes/performance.html]: ✅

File: bedrock/firefox/templates/firefox/developer/index.html (hunk 1 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/developer/index.html]: I updated it to http://localhost:8000/en-US/privacy/firefox/#pre-release which is the section that explains telemetry is collected differently in pre-release versions of Firefox.

File: bedrock/firefox/templates/firefox/developer/includes/highlights.html [hunk 6]
```diff
@@ -1,103 +1,88 @@
...
+      <p>
+      {% if l10n_has_tag('ember_reference_fix') %}
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/developer/includes/highlights.html]: Agree. Removed.

File: bedrock/firefox/templates/firefox/developer/includes/highlights.html [hunk 7]
```diff
@@ -1,103 +1,88 @@
...
-      <h2 class="section-title"><span>{{ _('Faster Information') }}</span></h2>
-
-      <div class="section-body">
-        <a rel="external" href="https://developer.mozilla.org/en-US/docs/Tools/Page_Inspector/How_to/Edit_fonts">
-          <h3 class="body-title">{{ _('Fonts Panel') }}</h3>
-
-          <img class="hero-image" src="{{ static('img/firefox/developer/hero-fonts.gif') }}" alt="">
-        </a>
-        <p>
+    {% if l10n_has_tag('new_features_20180830') %}
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/developer/includes/highlights.html]: I'm going to leave this one because I want this merged before I go on PTO.

**Comment by stephaniehobson** [comment]
Ready for re-review.

File: bedrock/firefox/templates/firefox/developer/includes/highlights.html (hunk 7 above)
[the remark this answers is not in the document]
Comment (craigcook) [reply bedrock/firefox/templates/firefox/developer/includes/highlights.html]: Agreed, it's not hurting anything to leave it.

**Pull request merged**